src/mpi/coll/ibcast/ibcast.c

```diff
@@ -108,11 +108,11 @@ int MPIR_Ibcast_sched_intra(void *buffer, int count, MPI_Datatype datatype, int
     else /* (nbytes >= MPIR_CVAR_BCAST_SHORT_MSG_SIZE) && (comm_size >= MPIR_CVAR_BCAST_MIN_PROCS) */
     {
         if ((nbytes < MPIR_CVAR_BCAST_LONG_MSG_SIZE) && (MPIU_is_pof2(comm_size, NULL))) {
-            mpi_errno = MPII_Ibcast_scatter_rec_dbl_allgather_sched(buffer, count, datatype, root, comm_ptr, s);
+            mpi_errno = MPIR_Ibcast_sched_intra_scatter_recursive_doubling_allgather(buffer, count, datatype, root, comm_ptr, s);
```

Comment:
If we're going to convert these to the namespace, should they not also be added to the selection algorithm (CVAR, enum, etc.)?